tests/nodes_data_conf_single_node.json

```diff
@@ -1,25 +0,0 @@
-{
```

Comment:
Why was this configuration file of load balancer lurking at the top-level of the **tests** directory?

Reply:
When the structure was changed, Travis was still running on a single node setup. This file was meant to be replaced before the Travis testing in place of the 5 node configuration file that exists in deploy/configs/load_balancer. Now that we have shifted to five node setup on Travis as well, we do not require this configuration file.